packages/compass-indexes/src/components/indexes-table/key-list.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  spacing,
+  css,
+  Badge,
+  BadgeVariant,
+  IndexIcon,
+} from '@mongodb-js/compass-components';
+
+import type { IndexDefinition } from '../../modules/indexes';
+
+const keyListStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const badgeStyles = css({
+  gap: spacing[1],
+});
+
+type KeyListProps = {
+  keys: ReturnType<IndexDefinition['fields']['serialize']>;
+};
+
+const KeyList: React.FunctionComponent<KeyListProps> = ({ keys }) => {
+  return (
+    <div className={keyListStyles} role="list">
+      {keys.map(({ field, value }) => (
+        <Badge
+          key={field}
+          data-testid={`${field}-key`}
+          variant={BadgeVariant.LightGray}
+          className={badgeStyles}
+          role="listitem"
+        >
+          {field}
+          <IndexIcon direction={value} />
```

Comment:
[unrelated nit feel free to ignore] Something I noticed about the index icon direction is that there is no `title` attribute. It might be a slightly nicer ux to provide that so folks can be know the terminology behind `ascending`/`descending`. Raising this since we're doing some other improvements to this view in this pr already.

Reply:
i tried to add the title but nothing happens, something with the icon component I suspect